### ISSUE TITLE
Roadmap

### ISSUE THREAD

**Issue by icechen1** [factorialLabs/beam-server#13]

**Comment by Deon** [comment]
Agreed, one of the other ideas we had earlier was rate-limiting, as well as Do-not-disturb/Offline modes, which history should tie with.

We should keep track of these ideas once finalized in a project or a simple TODO list.

**Comment by icechen1** [comment]
Yup, we can put it out as an issue here or use the project board? 

Another thing we should set up once we get the basic server functionality done is some sort of staging server on Azure VM.
